lighthouse-core/audits/critical-request-chains.js

```diff
@@ -98,44 +98,48 @@ class CriticalRequestChains extends Audit {
    * @return {!AuditResult} The score from the audit, ranging from 0-100.
    */
   static audit(artifacts) {
-    const networkRecords = artifacts.networkRecords[Audit.DEFAULT_PASS];
-    return artifacts.requestCriticalRequestChains(networkRecords).then(chains => {
-      let chainCount = 0;
-      function walk(node, depth) {
-        const children = Object.keys(node);
+    const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    return Promise.resolve().then(_ =>
```

Comment:
is there a reason to wrap this in a Promise.resolve? what can fail before we get into the `.then` of `requestNetworkRecords`?